src/api/services/system/dotcms-config.ts

```diff
@@ -24,14 +25,14 @@ export class DotcmsConfig extends CoreWebService {
     constructor(apiRoot: ApiRoot, http: Http) {
         super(apiRoot, http);
         this.configUrl = 'v1/appconfiguration';
-        this.getConfig();
     }
 
-    getConfig() {
-        this.requestView({
+    // TODO: try to make this a promise
+    getConfig(): Observable {
```

Comment:
We should create a new interface for it, and do : Obsevable<new_interface>
Also configParams's type should be the new one.